### ISSUE TITLE
Both v18 and v18HF fail to start after update

### ISSUE THREAD

**Issue by Simmons928** [LunaPixelStudios/Better-Minecraft#195]
### Describe the Bug

Updated all mods in modpack and server crashes on launch due to fabric verson incompatible with mods. 
Update Fabric version to 0.13.3 and server still fails to launch with different error. 

### Reproduction Steps

1.Update Server from v17 to v18 or v18HF
2.Attempt to start server normally
3.Server Fails to start properly

### Screenshots and Videos

After Updating both Modpack and Fabric
https://pastebin.com/nVCGVZ3i

Console Output After updating to v18HF but without updating fabric. 
https://pastebin.com/sQ76ZiL8

Provided Log Below is the Fabric Loader's .log file. 

### Crash Report or Log

https://pastebin.com/EmQGUBtr

### Operating System

Unraid 6.9.2

### Minecraft Version

1.18

### Modloader

Fabric

### Modpack Version

v18 HF

### Optifine

No

### Other Mods

Removed all non-pack mods and retest gives same result.

**Comment by SharkieTV** [comment]
Redownload the Serverpack, I reuploaded a fix

**Comment by Simmons928** [comment]
I redownloaded the mod pack, still getting dependency errors.  
https://pastebin.com/k1ibP3px

**Comment by SharkieTV** [comment]
You definitely didnt download the right serverpack.
https://www.curseforge.com/minecraft/modpacks/better-minecraft-fabric/files/3684116

**Comment by Simmons928** [comment]
Sharkie, that is what I downloaded and installed on my server. 
I downloaded v18HF, which is why I mentioned that version in the title. 
I downloaded it last night, 4-5 hours after you uploaded it. I attempted the update, it failed in the same way as v18. 
You told me to redownload it, which I did. 
I went to this page, and downloaded the same v18HF file AGAIN. 
Attempted the install, it fail in the same way. 

This is the link I downloaded the server file for v18HF
https://www.curseforge.com/minecraft/modpacks/better-minecraft-fabric/files

The link you provided's MD5 is: e4ef6ae1ec9127178b74b663ae48e40e
The link I went to shows the same MD5: e4ef6ae1ec9127178b74b663ae48e40e

**Comment by Simmons928** [comment]
To Humor you, I used your link and downloaded it again. 
Below is the result, it is still broken. 
https://pastebin.com/ZQ995Gbh

**Comment by SharkieTV** [comment]
Not sure what to tell you, others aren't having this issue nor am I, I can only assume you are unzipping the wrong files and using the wrong ones.

In the server files when I download it, it installs 0.13.3 and in your pastebin its 0.13.2 so the log still tells me you are using the wrong files. Not much I can do here as it still suggests you have the wrong files. All I can tell you to do is delete all olds files you have, make a video of you downloading the correct files, after unzipping it right click the start.bat and shop me whats in it and start the server.

**Comment by Simmons928** [comment]
"...right click the start.bat..." Oh....
I don't use your start.bat file. My sever management tool handles all that on its own.
I did run it on a separate machine and move the files to the server and it worked. 
What is weird, is I manually installed fabric 0.13.3 with the mods, config data, and scripts from the HF zip and I got a very strange error. (This here https://pastebin.com/nVCGVZ3i)

I will be the first to tell you I am an idiot.
However, the patch notes did not mention that fabric was also updated. There for I assumed that 0.13.2 was the correct Especially following the error I received after manually updating it.

Takeaways for me, I will use your start.bat to get all the files for an update in the future. 
Suggestion for you, if you change the fabric version, put it in the patch notes.
